app/components/insights-privacy-selector.js

```diff
@@ -0,0 +1,36 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+export const INSIGHTS_PRIVACY_OPTIONS = {
+  PUBLIC: 'public builds',
+  PRIVATE: 'public and private builds',
+};
+
+export default Component.extend({
+  tagName: 'span',
+  classNames: ['insights-privacy-selector'],
+  classNameBindings: ['isPrivateViewable:insights-privacy-selector--selectable'],
+
+  isPrivateViewable: false,
+  includePrivate: false,
+  showFrame: false,
+
+  availableOptions: computed('isPrivateViewable', function () {
+    if (!this.isPrivateViewable) {
+      return [];
+    }
+    return Object.values(INSIGHTS_PRIVACY_OPTIONS);
```

Comment:
I would make it a single line:
```javascript
return this.isPrivateViewable ? Object.values(INSIGHTS_PRIVACY_OPTIONS) : [];
```
It's totally optional and opinionated. I don't know, I just like shorter 😄

Reply:
I like it! Change made 😄